--- a/target/linux/ramips/image/mt7621.mk
+++ b/target/linux/ramips/image/mt7621.mk
@@ -1792,7 +1792,7 @@ define Device/jdcloud_re-sp-01b
   IMAGE_SIZE := 27328k
   DEVICE_VENDOR := JDCloud
   DEVICE_MODEL := RE-SP-01B
-  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e \
+  DEVICE_PACKAGES := mmc-utils kmod-fs-ext4 kmod-mt7603 kmod-mt7615e \
 	kmod-mt7615-firmware kmod-mmc-mtk kmod-usb3
 endef
 TARGET_DEVICES += jdcloud_re-sp-01b
